controller/tests/server/test_route_processing.py

```diff
@@ -1283,31 +1283,6 @@ def test_set_protocols__waits_for_stim_info_in_shared_values_dict_to_be_updated_
     assert mocked_sleep.call_args_list == [mocker.call(0.1), mocker.call(0.1)]
 
 
-def test_after_request__redacts_mantarray_nicknames_from_system_status_log_message(
```

Comment:
Why did we remove this test?

Reply:
This test checks that `systems status` logs are redacted if the status code is 200. If we are no longer logging 200 status code `system_status` logs, but instead the error , the logs look completely different and don't contain the message to redact? So I removed this test and replaced it with two tests, one to check logs on a 200 code and one to check logs on a none 200 status code.

  ```
if response.status_code == 200 and "system_status" not in rule.rule:
        # Tanner (1/19/21): using json.dumps instead of an f-string here allows us to perform better testing of our log messages by loading the json string to a python dict
        msg += json.dumps(response_json)
        logger.info(msg)
    elif response.status_code != 200:
        msg += response.status
        logger.info(msg)
    return response
```

Reply:
@tannermpeterson am I understanding the test incorrectly?

Reply:
I believe you understanding it correctly. I'm just asking a question to make sure that the test is no longer necessary